.github/scripts/lintrunner.sh

```diff
@@ -30,6 +30,9 @@ python3 -m tools.pyi.gen_pyi \
     --tags-path aten/src/ATen/native/tags.yaml \
     --deprecated-functions-path "tools/autograd/deprecated.yaml"
 
+# lintrunner will fail if the tee file exists
+rm -f lint.json
+
```

Comment:
This sounds suspicious, how did it work all that time beforehand?

Reply:
Most of the time it's fine because we run in a clean VM. But I had ssh'd to the CI machine debug to an issue and it had a stale file (and had really weird errors running this script since it failed but lintrunner said "no errors")